tests/e2e/env/config/default.json

```diff
@@ -1,5 +1,5 @@
 {
-  "url": "http://localhost:8084/",
+  "url": "http://localhost:8888/",
```

Comment:
Btw, the default test instance is running on `8889`, which has some options more optimized for tests. (in the future, for example, script debug will be set to false there by default)